$OpenBSD: patch-notmuch-new_c,v 1.1.1.1 2020/05/15 08:58:14 sthen Exp $

when upgrading database, back up the old db without compression since
--gzip is not supported in this port

Index: notmuch-new.c
--- notmuch-new.c.orig
+++ notmuch-new.c
@@ -1156,7 +1156,7 @@ notmuch_new_command (notmuch_config_t *config, int arg
 	     * relatively small. */
 
 	    const char *backup_name =
-		talloc_asprintf (notmuch, "%s/dump-%04d%02d%02dT%02d%02d%02d.gz",
+		talloc_asprintf (notmuch, "%s/dump-%04d%02d%02dT%02d%02d%02d",
 				 dot_notmuch_path,
 				 gm_time->tm_year + 1900,
 				 gm_time->tm_mon + 1,
@@ -1168,11 +1168,11 @@ notmuch_new_command (notmuch_config_t *config, int arg
 	    if (add_files_state.verbosity >= VERBOSITY_NORMAL) {
 		printf ("Welcome to a new version of notmuch! Your database will now be upgraded.\n");
 		printf ("This process is safe to interrupt.\n");
-		printf ("Backing up tags to %s...\n", backup_name);
+		printf ("Backing up tags to %s (uncompressed)...\n", backup_name);
 	    }
 
 	    if (notmuch_database_dump (notmuch, backup_name, "",
-				       DUMP_FORMAT_BATCH_TAG, DUMP_INCLUDE_DEFAULT, true)) {
+				       DUMP_FORMAT_BATCH_TAG, DUMP_INCLUDE_DEFAULT, false)) {
 		fprintf (stderr, "Backup failed. Aborting upgrade.");
 		return EXIT_FAILURE;
 	    }
